src/scf-release/src/acceptance-tests-brain/test-scripts/011_nfspersi_test.rb

```diff
@@ -50,7 +60,15 @@
     run "kubectl delete -n #{NS} -f #{SKUBEC}"
 end
 run "kubectl create -n #{NS} -f #{SKUBEC}"
-wait_for_namespace NS
+
+
+# Do not run the wait below into the full timeout for the test
+# case. We wish to do some inspection and proper cleanup when
+# we abort.  So we wrap a timeout so that we get two minutes to
+# do any cleanup we need.
+Timeout::timeout(ENV.fetch('TESTBRAIN_TIMEOUT', '600').to_i - 120) do
+  wait_for_statefulset(NS, "nfs-test-server")
```

Comment:
We've been using 4-space indentation for Ruby.

Reply:
Will fix. Running on emacs defaults.

Reply:
Done with commit 4d305948ad6992258efd08bdeda61d6ee3e81ab1